src/app/view/components/dot-toolbar/dot-toolbar.component.ts

```diff
@@ -33,7 +35,9 @@ export class ToolbarComponent implements OnInit {
 
     siteChange(site: Site): void {
```

Comment:
So if I'm in "site browser" and I'm seeing like 20 folders and I change the site, do the site browser update to reflect the folders for the new site? (the same goes with all the portlets)

Reply:
yes it is updated